bundle/edu.gemini.shared.gui/src/main/java/edu/gemini/shared/gui/calendar/LunarDateSymbolSupplier.java

```diff
@@ -128,20 +105,20 @@ public int compareTo(Object o) {
 
         int _yearCount;
 
-        List _years;  // each entry is a List of Months
+        List<List<List<DateSymbol>>> _years;  // each entry is a List of Months
```

Comment:
Ugh. How horrible this is!
Having it properly typed, though, will of course make it much easier to understand what is going on.